CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+When contributing to Kubeapps, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
```

Comment:
Let's remove email and other methods. Let's just say that people should open an issue and discuss it there.

Reply:
done